content/docs/user-guide/how-to/troubleshooting.md

```diff
@@ -1,6 +1,7 @@
-# Troubleshooting
```

Comment:
this should be part of the regular User Guide ... it's well established section (like FAQ or How to, etc). It's strange to see it inside

Reply:
OK. Rolled back this change in 1a6948ea.